### PR TITLE
Pt 162578740 fate implementation spike

### PR DESCRIPTION
**Pull request by happi**

### CODE REVIEW

**Comment by lucafavatella** [comment]
I am taking the liberty of tracking as WIP for easing filtering of PRs re minerva.

**Comment by lucafavatella** [comment]
Temporarily labelling as WIP for easing filtering PRs while releasing.

File: apps/aefate/rebar.config [hunk 1]
```diff
@@ -0,0 +1,33 @@
+{minimum_otp_vsn, "20.1"}.
+
+{erl_opts, [debug_info]}.
+
+{escript_incl_apps, [aefate, aebytecode, lager, getopt]}.
+{escript_main_app, aefate}.
+{escript_name, aefate}.
```
Comment (lucafavatella) [inline apps/aefate/rebar.config]: Is there a smoke test for this, hooked to CI?

Comment (happi) [reply apps/aefate/rebar.config]: No. The stand alone engine is only used for debugging now. We might use it for some tests in the future and then this build will be tested.

**Pull request merged**